robot/Cumulus/resources/NPSP.py

```diff
@@ -635,19 +636,19 @@ def click_save(self, page):
             id="j_id0:theForm:j_id9:j_id10:saveBTN"
             locator = npsp_lex_locators["id"].format(id)
             self.selenium.get_webelement(locator).click()
-     
+
     def enter_payment_schedule(self, *args):
-        """Enter values into corresponding fields in Levels page"""                 
+        """Enter values into corresponding fields in Levels page"""
         #if name == "Payments":
         #id = ["paymentCount","intervals","intervalunits"]
         id = ["paymentCount","vfForm:intervalnumber","intervalunits"]
         for i in range(len(args)):
             locator = npsp_lex_locators['id'].format(id[i])
             loc = self.selenium.get_webelement(locator)
-            self.selenium.set_focus_to_element(locator)       
+            self.selenium.set_focus_to_element(locator)
             self.selenium.select_from_list_by_label(loc,args[i])
-            time.sleep(2)    
-                
+            time.sleep(2)
+
     def verify_payment_split(self, amount, no_payments):
         loc = "//*[@id='pmtTable']/tbody/tr/td[2]/div//input[@value= '{}']"
```

Comment:
Are you sure you want to rely on indexing here? Is there any other way you can find this locator? either by finding the relationship with the parent locator(following,following-sibling, descendant, child)And please move the locator to the locators file.

Reply:
Modified it to be better. BTW, The above and this code is not a new addition to the NPSP.py file. The code is preexisting.